freqtrade/rpc/rpc.py

```diff
@@ -1030,3 +1031,11 @@ def _rpc_sysinfo() -> Dict[str, Any]:
             "cpu_pct": psutil.cpu_percent(interval=1, percpu=True),
             "ram_pct": psutil.virtual_memory().percent
         }
+
+    def _health(self) -> Dict[str, str]:
+        last_p = self._freqtrade.last_process
+        return {
+            'last_process': str(last_p),
+            'last_process_loc': last_p.astimezone(tzlocal()).strftime(DATETIME_PRINT_FORMAT),
```

Comment:
I've now made some changes to this ...
especially this part can be argued ... this will move the timestamp to "local time" (in server timezone) ...

Reply:
Thanks, been buzy with life/work, I can sort this out in the weekend though.

Reply:
I've pushed "something" already (i'm sometimes faster fixing stuff than commenting "please fix" :laughing: )

 ... this is more a "do we want it this way" question